src/test/scala/cromwell/WdlExpressionSpec.scala

```diff
@@ -24,11 +25,11 @@ class WdlExpressionSpec extends FlatSpec with Matchers {
   }
   class TestFunctions extends WdlFunctions {
     def getFunction(name: String): WdlFunction = {
-      def b(params: Seq[WdlValue]): WdlValue = {
-        WdlInteger(params.head.asInstanceOf[WdlInteger].value + 1)
+      def b(params: Seq[Try[WdlValue]]): Try[WdlValue] = {
+        Success(WdlInteger(params.head.asInstanceOf[Try[WdlInteger]].get.value + 1))
```

Comment:
Yeah, something to discuss.  It is a bit noisy to be sure.  At the actor level I do appreciate being compelled by the Try[T] signature at top-level business logic to think about whether an operation succeeded or not, since this decides whether the original sender is messaged for success or failure.

Use of Try at the highest level doesn't necessarily drive it all the way down to the bottommost layers, e.g. I punted pretty hard on Try in command evaluation.  :smile:  Maybe now with some more practice I could do that better, but that's probably not the most valuable thing I could be working on at the moment...